peachjam/templates/peachjam/layouts/main.html

```diff
@@ -12,6 +12,16 @@
   <!--  app-prod.js depends on pdf.js to be loaded-->
   <script defer src="{% static 'js/app-prod.js' %}"></script>
 
+ <!-- Small script to add the current user agent to all elements with the class name 'pocketlaw-hidden'. -->
+  <script type="text/javascript">
+    document.addEventListener("DOMContentLoaded", function () {
+      const pocketlawElements = document.querySelectorAll('.pocketlaw-hidden');
+      pocketlawElements.forEach(el => {
```

Comment:
Rather set the user agent on the html element. Then we'll use css selectors to hide/show content. 

That way the JavaScript doesn't need to know about any special selectors - literally all it does is set the user agent data attribute on html. Everything else happens in css